---
title: "Why are losses of PRDM9 tolerated?"
format:
  md:
    variant: gfm
---

### Background

Recombination, i.e., the repair of programmed double strand breaks via exchange of genetic information between homologous chromosomes, is a key step in meiosis and is thought to provide fuel for the evolution of sexual reproduction: shuffling of genetic variation. Yet, recombination varies wildly across the Tree of Life. One way is in the manner in which double strand breaks and crossovers to resolve those breaks are localized on the genome. In many species, recombination "hotspots" are determined by a gene called PRDM9. While there are many species that have hotspots and make do in meiosis without it, PRDM9 appears to be essential in others. In fact, in mice, it has been [shown](https://www.nature.com/articles/nature16931) that divergence in the hotspots encoded by the zinc finger domain of PRDM9 in _Mus musculus_ and _Mus musculus domesticus_ underlies  infertility in hybrid males. (The problem in the hybrids is with homology search, i.e., how chromosomes find each other in the three dimensional soup that is the nucleus. Chromosomes in hybrid sperm pair up, more or less, at random when the PRDM9 alleles in the parents are brought onto the same background.) In other words, evolution of the zinc finger domain of PRDM9 appears to be _the_ reason that speciation of _musculus_ and _domesticus_ is essentially complete!

These data support the idea that PRDM9 is essential player in male meiosis, but comparative analyses of fine-scale crossover patterning have shown that there are many clades (e.g., [birds](https://www.science.org/doi/10.1126/science.aad0843)) where PRDM9 is non-functional. In these species, there are still recombination hotspots. But they appear at location that have little to do with the identity of PRDM9 (when the gene is around) and determined, largely, by the accessibility of DNA. Birds have lost the gene, dogs no longer use it, and many fish have some parts but are missing others. (Interestingly, losses of a complete PRDM9 appear to concide with changes in the distribution of intron sizes?!?)

__What explains losses of PRDM9?__

```{r}
```