---
layout: page
title: "Research"
permalink: /research/
main_nav: true
---

**I use mechanistic mathematical models, genomic data, and computational statistical tools to answer biological questions.** 

<div style="margin: 40px 0;">
  <img src="/assets/bear.JPG" style="width: 100%; max-width: none;">
</div>

<br>

One of the subjects I am most interested in right now is the **recombination rate variation**[^1].

- A particularly salient way recombination varies is along genomes. The landscape (i.e., probability distribution) of recombination varies between gametes, individuals, sexes, populations, and species[^2] and appears to be quite evolutionarily liable. A wealth of data (e.g., from single sperm sequencing in humans and other species) makes it an exciting time to think about variation in recombination at these scales.

- One reason it is difficult to understand the causes and consequences of variation in recombination on the genome is that the phenotype is effectively infinite-dimensional. The recombination landscape on the chromosome is best thought of as a function of genomic location, with co-variances at all pairs of locations that constrain how it can evolve. With Puneeth Deraje, I've been working on a mechanistic model that describes how this function evolves under neutrality (i.e., drift and cis modifier accumulation). We have shown that, in a finite population, the mean CDF of recombination follows a certain Gaussian process. We are currently using comparative data to make inferences (under this model) about the tempo and mode of landscape evolution, i.e., the manner in which modifiers accumulate.

<br>

My other main scientific interest is **virus evolution and emergence**. RNA viruses are so rapidly evolving that, by conventional metrics of the rate of molecular evolution, their genomes should turnover _neutrally_ in $\sim 10^3$ years. By some estimates, the rate of recombination (during co-infection of the same host) is even greater. Isn't that remarkable! I am interested in understanding how viral mutation and recombination shape the propensity of the pathogen to successfully spillover and sustain transmission (i.e., emerge) in humans[^3]. I am also very interested in how, as phenotypes, viral mutation and recombination rates vary and evolve.

- Although we know recombination and reassortment are common in RNA viruses and of concern for human health, the _evolution_ of these process is not particularly well-understood. An especially interesting alternative to the idea reassortment evolved to shuffle genetic material is that it is a necessary byproduct of the segmented nature of some viral genomes, which evolved because it allows finer control over gene expression. Weighing the shuffling and gene expression control hypotheses is a problem I am very interested in.

<br>

[^1]: A great introduction to the subject is [this paper](https://www.nature.com/articles/s41576-020-0240-1) by PeÃ±alba and Wolf.

[^2]: In fact, there is a remarkable experiment showing that the gene, PRDM9, that determines where recombination occurs on the genome (i.e., the set of recombination _hotspots_) in many vertebrates underlies the formation of different species to begin with ðŸ˜±.

[^3]: And there happens to be a paper to this end [on bioRxiv](https://www.biorxiv.org/content/10.1101/2025.10.01.679867v1)!